test/test.py

```diff
@@ -506,6 +506,41 @@ def test_display_completions(self):
         self.assertEqual("ttt", disp.get("--oh", ""))
         self.assertEqual("ccc", disp.get("--ch", ""))
 
+        if not USING_PYTHON2:
```

Comment:
Since this doesn't share any code with the rest of the test, how about making it a separate test and adding e.g. `@skipIf(USING_PYTHON2, 'Subparser aliases aren't supported on Python 2')`?